# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipw2100/nodepmod.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

If we run depmod than with version argument - however the linux package
does it at the end - so we skip it at all ,-)

--- ipw2100-1.0.5/Makefile.orig	2005-03-25 15:24:40.809653152 +0100
+++ ipw2100-1.0.5/Makefile	2005-03-25 15:25:58.928777240 +0100
@@ -205,7 +205,6 @@
 install: modules
 	install -d $(KMISC)
 	install -m 644 -c $(addsuffix .ko,$(list-m)) $(KMISC)
-	/sbin/depmod -a
 	@echo "Don't forget to copy firmware to your hotplug's firmware direectory and have the "
 	@echo "hotplug tools in place."
 	@echo "See INSTALL for more information."
 
 
